tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Testing
+
+*Unittest is the preferred test framework for this project*
+
+## Run all tests
+
+From the top directory:
+
+```
+python -m unittest discover -vv -s tests -t $PWD
```

Comment:
Hi @tompscanlan ,

Did you try executing this ?  We'll get Not a directory error.
```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/__main__.py", line 12, in <module>
    main(module=None)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/main.py", line 94, in __init__
    self.parseArgs(argv)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/main.py", line 113, in parseArgs
    self._do_discovery(argv[2:])
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/main.py", line 214, in _do_discovery
    self.test = loader.discover(start_dir, pattern, top_level_dir)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 206, in discover
    tests = list(self._find_tests(start_dir, pattern))
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 241, in _find_tests
    paths = os.listdir(start_dir)
OSError: [Errno 20] Not a directory: '/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/liota-0.2-py2.7.egg/tests'
```

However, this works:
```
sudo python -m unittest discover
```